index.js

```diff
@@ -135,14 +135,13 @@ module.exports = function(tilelive, options) {
               return next(err);
             }
 
-            return async.waterfall([
-              async.apply(im.encode, "webp"),
-              function(img, done) {
-                vt.addImage(img, src.info.id.replace(/\./g, "_"));
```

Comment:
It looks like the cleaner change is just to change `vt.addImage` to `vt.addImageBufferSync`, as the `webp` encode + error handling was taken care of by `async.waterfall`.